src/utils.ts

```diff
@@ -150,6 +150,116 @@ export function literal2Asm(l: string): [string, string] {
   throw new Error(`<${l}> cannot be cast to ASM format, only sCrypt native types supported`);
 }
 
+
+
+
+/**
+ * convert literals to Scrypt Type
```

Comment:
Duplicate of https://github.com/scrypt-sv/scryptlib/blob/d6526fa03d7ed2acf14e3e146c545eb2cbc5b1f7/src/utils.ts#L58

Reply:
add a new function literalParser.